### PR TITLE
Update failure injection demos

### PR DESCRIPTION
**Pull request by yuanchen8911**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Update failure injection demos using the latest release 0.6.0. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
Update demos and docs
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: demo/container-failure.svg
Comment (wzshiming) [inline demo/container-failure.svg:1]: This demo seems to be failure, it need to regenerate

File: site/static/charts/index.yaml
Comment (wzshiming) [inline site/static/charts/index.yaml:1]: This is helm chart index.yaml, revert this

Comment (yuanchen8911) [reply site/static/charts/index.yaml:1]: Are you suggesting to revert the changes to `index.yaml` and keep `v0.5.2`?

File: demo/container-failure.svg
Comment (yuanchen8911) [reply demo/container-failure.svg:1]: There's an error in the stage file. Can you take a look? @wzshiming  
```
~/sigs.k8s.io/kwok$ kubectl apply -f https://github.com/kubernetes-sigs/kwok/raw/main/kustomize/stage/pod/chaos/pod-container-running-failed.yaml
Error from server (BadRequest): error when creating "https://github.com/kubernetes-sigs/kwok/raw/main/kustomize/stage/pod/chaos/pod-container-running-failed.yaml": Stage in version "v1alpha1" cannot be handled as a Stage: strict decoding error: unknown field "spec.weightFrom"
```

File: site/static/charts/index.yaml
Comment (yuanchen8911) [reply site/static/charts/index.yaml:1]: reverted the changes

File: demo/container-failure.svg
Comment (yuanchen8911) [reply demo/container-failure.svg:1]: Fixed it and regenerated the demo.

File: demo/container-failure.demo [hunk 1]
```diff
@@ -3,8 +3,9 @@
 # Create a cluster with kwok 
 kwokctl create cluster --enable-crds Stage
 
-# Apply stage files
-kubectl apply -f https://github.com/kubernetes-sigs/kwok/releases/download/v0.5.2/stage-fast.yaml
+# Deploy kwok crds and stage files 
+kubectl apply -f https://github.com/kubernetes-sigs/kwok/releases/download/v0.6.0/kwok.yaml
```
Comment (wzshiming) [inline demo/container-failure.demo:7]: this cluster is created by kwokctl, that don't need this kwok

**Pull request closed without merging**